tests/index.py

```
@@ -0,0 +1,450 @@
+import os
+from api import *
```

Ruff (0.8.2)

`from api import *` used; unable to detect undefined names

(F403)